pkg/cmd/run.go

```diff
@@ -680,7 +679,6 @@ func init() {
 	runCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
 	runCmd.Flags().Bool("no-exit-on-write-failure", false, "do not exit if unable to write the fallback file")
 	runCmd.Flags().Bool("forward-signals", forwardSignals, "forward signals to the child process (defaults to false when STDOUT is a TTY)")
-	// secrets mount flags
```

Comment:
I see now that the value of these comments was just for grouping the flags, rather than explaining what they do. They might be worth keeping around.

Reply:
Deleting the corresponding `chore:` commit that deleted them!